docker/environment.yml

```diff
@@ -22,11 +36,11 @@ dependencies:
   - descartes
```

Comment:
What needs `descartes`? This library is ancient.

Reply:
> What needs `descartes`? This library is ancient.

https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/commit/90ccbece93469c3783e8d786592ac92e7fdf5f15

I just update the PR description with list of package changes.  You might want to review the PR description again.